app/components/appMainModule/committeePageModule/state.js

```diff
@@ -62,24 +47,8 @@ module.exports = function($stateProvider) {
       parent: 'appMain.committee.main'
     },
     resolve: {
-      contributors: function($stateParams, $q) {
-        return $q.resolve([
-          {
-            name: 'Samantha Brooks',
-            amount: 700,
-            date: new Date('2015-04-12')
-          },
-          {
-            name: 'Lisa Sheppards',
-            amount: 700,
-            date: new Date('2015-01-13')
-          },
-          {
-            name: 'Raoul Esponsito',
-            amount: 700,
-            date: new Date('2015-04-04')
-          }
-        ]);
+      contributors: function($stateParams, disclosureApi) {
+        return disclosureApi.committee.get();
```

Comment:
This should be a contributors endpoint, passing the committee_id, do you have that stubbed on the backend?